### ISSUE TITLE
[4] badges underlined in Site info dashboard module

### ISSUE THREAD

**Issue by PhilETaylor** [joomla/joomla-cms#34137]
### Steps to reproduce the issue

publish the Site Information module in home dashboard

### Expected result

badges with the numbers should be like other badges and not underlined (unless accessibility options to underline links is enabled) 

### Actual result

By default:
<img width="645" alt="Screenshot 2021-05-23 at 16 08 55" src="https://user-images.githubusercontent.com/400092/119266080-3cfa3b80-bbe1-11eb-81dc-672e43c94688.png">

### System information (as much as possible)

google chrome on mac. safari on mac

**Comment by chmst** [comment]
Closing as we have a PR. Please test #34142
